autoload/jedi.vim

```diff
@@ -403,7 +403,7 @@ function! jedi#configure_call_signatures()
         autocmd InsertEnter <buffer> let g:jedi#first_col = s:save_first_col()
     endif
     autocmd InsertEnter <buffer> let s:show_call_signatures_last = [0, 0, '']
-    autocmd InsertLeave <buffer> PythonJedi jedi_vim.clear_call_signatures()
```

Comment:
I had removed it intentionally, but now added it back - I think it's better to be explicit on resetting/invalidating the cache, although it gets done on `InsertEnter` now, too.